pyrekordbox/config.py

```diff
@@ -274,11 +274,6 @@ def _get_rb_config(
         # `major_version` is compared to the version string
         rb_version = application_dirname.replace("rekordbox", "").strip()
         rb_version = rb_version.replace(".app", "")
```

Comment:
Ok there is at least one issue with my fix: So here you are trying to extract a version number from this arbitrary directory name, which you later put into to the final conf dictionary. Correct? Where will this version number be used again? Why is it important? _If_ it is important, are there other (better) ways to export this version number?